modules/migration-updating-internal-images.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * migration/migrating_3_4/migrating-applications-with-cam-3-4.adoc
+
+[id='migration-updating-internal-images_{context}']
+= Updating internal images manually
+
+If your application uses images from the `openshift` namespace, the required versions of the images must be present on the target cluster.
+
+If the {product-title} 3 image is deprecated in {product-title} {product-version}, you can manually update the image stream tag using Podman.
+
+.Prerequisites
+
+* You must have Podman installed.
+* You must have `cluster-admin` privileges.
+
+.Procedure
+
+. Expose the internal registries on the source and target clusters.
+. If you are using insecure registries, add your registry host values to the `[registries.insecure]` section of `/etc/container/registries.conf` to ensure that Podman does not encounter a TLS verification error.
```

Comment:
I think this step needs an example.
On which host is the `/etc/container/registries.conf` file?